pvsite_datamodel/sqlmodels.py

```diff
@@ -122,6 +122,9 @@ class SiteSQL(Base, CreatedMixin):
     """
 
     __tablename__ = "sites"
+    __table_args__ = (
+        UniqueConstraint("client_site_name", "ml_id", name="uniq_cons_client_name_ml_id"),
+    )
```

Comment:
I'd probably remove this for the moment, once #148 is done, we can add something like this in